lib/animina_web/components/bookmark/bookmark_component.ex

```diff
@@ -65,6 +76,13 @@ defmodule AniminaWeb.BookmarkComponent do
         language={@language}
         reason={@reason}
         current_user={@current_user}
+        state={
+          if @current_user.profile_photo && is_atom(@current_user.profile_photo.state) do
```

Comment:
Don't we have a logic problem here? What if there is no photo? Than make_sure_photo_state_is_atom is called. Is that correct?